src/transformStateWithClones.js

```diff
@@ -7,7 +7,42 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arrayOfStates = [];
+  const stateCopy = Object.assign({}, state);
+  const addPropertiesAction = 'addProperties';
```

Comment:
```suggestion
  const ADD_ACTION = 'addProperties';
```
Same with other static variables